kie-server-parent/kie-server-tests/kie-server-integ-tests-optaplanner/src/test/java/org/kie/server/integrationtests/optaplanner/OptaplannerIntegrationTest.java

```diff
@@ -251,10 +266,20 @@ public void testGetSolvers() {
         assertNull(returnedInstance.getScoreWrapper().toScore());
     }
 
-    @Test
+    @Test(timeout = 15_000)
     public void testExecuteSolver() throws Exception {
+        testExecuteSolverHelper(SOLVER_1_ID);
+    }
+
+    @Test(timeout = 15_000)
```

Comment:
Suggestion: Usually we take 60_000 because some jenkins machine are extremely slow